src/frontend/style/grid.css

```diff
@@ -0,0 +1,689 @@
+
+/*  SECTIONS  ============================================================================= */
```

Comment:
Where is this code from?  Did you write all this from scratch?  If not, we need a license.

Reply:
I'm not sure about the license. The code is from http://www.responsivegridsystem.com/
![image](https://user-images.githubusercontent.com/34637673/68987972-febad980-07fd-11ea-94ca-3da68a03953a.png) 

Under the [Creative Commons Attribution 3.0 License](https://creativecommons.org/licenses/by/3.0/). If this license may cause an issue, I will replace the code with the bootstrap CDN link.

Unfortunately, the responsive grid system does not support the CDN link.